pkg/relay/ethereum/findItems_test.go

```diff
@@ -0,0 +1,75 @@
+package ethereum
+
+import (
+	"testing"
+	"fmt"
+	"slices"
+)
+
+func TestFindItems(t *testing.T) {
+	cases := []struct{
+		size int
+		expect int
+		expectLog []int
+	}{
+		{
+			size: 0,
+			expect: 0,
+			expectLog: []int{ },
+		},
+		{
+			size: 10,
+			expect: 10,
+			expectLog: []int{ 10 },
+		},
+		{
+			size: 10,
+			expect: 9,
+			expectLog: []int{ 10, 5, 8, 9 },
+		},
+		{
+			size: 10,
+			expect: 0,
+			expectLog: []int{ 10, 5, 3, 2, 1 },
+		},
+		{
+			size: 10,
+			expect: 0,
+			expectLog: []int{ 10, 5, 3, 2, 1 },
+		},
```

Comment:
@dai1975 These two cases are same. Is it intentional?

Reply:
It may be mistake. deleted.